ntp-proto/src/peer.rs

```diff
@@ -9,6 +9,7 @@ use serde::{Deserialize, Serialize};
 use tracing::{debug, info, instrument, trace, warn};
 
 const MAX_STRATUM: u8 = 16;
+const POLL_WINDOW: std::time::Duration = std::time::Duration::from_secs(5);
```

Comment:
Maybe this is something that should be configurable, such that it can be set lower on high performance networks and higher on a mobile device on a slow connection?